pettingzoo/magent/adversarial_pursuit_v3.py

```diff
@@ -71,7 +71,11 @@ def get_config(map_size, minimap_mode, tag_penalty):
 
 
 class _parallel_env(magent_parallel_env, EzPickle):
-    metadata = {'render.modes': ['human', 'rgb_array'], 'name': "adversarial_pursuit_v3"}
+    metadata = {
+        "render.modes": ["human", "rgb_array"],
+        'name': "adversarial_pursuit_v3",
+        "video.frames_per_second": 5,
```

Comment:
Same here. Is this a suggestion for how fast it should be rendered?

Reply:
Yeap, this metadata thing is mostly a suggestion for how fast it should be rendered at.